labgrid/driver/sshdriver.py

```diff
@@ -104,7 +114,7 @@ def run(self, cmd):
         returns:
         (stdout, stderr, returncode)
         """
-        complete_cmd = "ssh -x {prefix} {user}@{host} {cmd}".format(
+        complete_cmd = "ssh -x{prefix} {user}@{host} {cmd}".format(
```

Comment:
As above I'm wondering about the change here.

Reply:
i found these changes necessary to prevent 2 spaces following each other in the cmd line.
(2 spaces cause split(' ') to add an empty element in the list which will be treated by ssh as the hostname...)

Reply:
Ah, I see. In that case I would like you too shuffle the `self.ssh_prefix += " -o LogLevel=ERROR"` line to the top and handle the extra missing space there. Otherwise we would have to touch the get and put functions as well. I tested your changes against a local device, works fine :)

Reply:
damn - i missed put and get.
should work now with one change less ;-)